src/js/view/view.js

```diff
@@ -340,6 +340,45 @@ function View(_api, _model) {
         playerViewModel.change('playlistItem', onPlaylistItem);
         // Triggering 'resize' resulting in player 'ready'
         _lastWidth = _lastHeight = null;
+
+        // Setup floating scroll handler
+        const topOffset = 62;
```

Comment:
i hate this but it's a magic number in the css too

Reply:
That magic number is supposed to account for site headers. But the new ticket suggests we should get rid of that (even though it could break mobile sites currently using this feature).